util/test/README.md

```diff
@@ -8,7 +8,14 @@ A lot of the tests rely on a 'demos' program which contains small self-contained
 
 To build on windows, open `demos.sln` and compile. There are no required external dependencies
 
-To build on linux, compile using cmake with `demos/CMakeLists.txt`. You'll need `libX11`, `libxcb`, and `libX11-xcb`. These are all needed to build RenderDoc with GL support so you likely have them already.
+To build on linux, compile using cmake with `demos/CMakeLists.txt` ie.
+
+```
+cmake demos/CMakeList.txt -B build
```

Comment:
`CMakeList.txt` has a typo so I don't think this command works as-is. Generally I pass cmake the directory (just `demos/` here) instead of the path to the `CMakeLists.txt`. The equivalent to the command in the main repo docs would be something like:

```
cmake -Bbuild -Hdemos
make -C build
```

I forget if a space between the `-B`/`-H` and the path works, I have a feeling it doesn't.